nemo_skills/pipeline/eval.py

```diff
@@ -214,6 +214,10 @@ def eval(
         "",
         help="Additional sbatch kwargs to pass to the job scheduler. Values should be provided as a JSON string or as a `dict` if invoking from code.",
     ),
+    extra_metrics_arguments: str = typer.Option(
```

Comment:
maybe we can call it extra_metrics_kwargs (or just metrics_kwargs)? To distinguish between arguments that are just string pass-through (called args/arguments) vs arguments that take in a dict (called kwargs)